pipeline_steps/common.groovy

```diff
@@ -904,4 +906,20 @@ void setTriggerVars(){
   print ("Trigger: ${env.RE_JOB_TRIGGER} (${env.RE_JOB_TRIGGER_DETAIL})")
 }
 
+// add wrappers that should be used for all jobs.
+// max log size is in MB
+void globalWraps(Closure body){
```

Comment:
Would it make sense to add another parameter for the timeout value, so that individual jobs can override on a case-by-case basis without having to add inner timeouts specific for the job?  If we do, I'd personally like to see the default (for when none specified) be far lower, like 1 hour.  This change would also allow us to remove some existing timeouts that are littered through jobs and common utils.

Reply:
I think this is a good suggestion. I'd prefer to see it done as a separate patch. `globalWraps` allows for additional layers to be added painlessly. The timestamp and log size pieces don't duplicate anything existing. I'd suggest fixing this code without including any timeout changes then creating a separate pull request to address all the occurrences of timeout.

Reply:
OK, Will do this as a separate patch.

Reply:
https://rpc-openstack.atlassian.net/browse/RE-1263